cdiff.py

```diff
@@ -491,14 +572,7 @@ def _fit_with_marker_mix(text, base_color, width, pad=False):
                     count += 1
                     text = text[1:]
 
-            if count == width and tag_re.sub('', text):
-                # Was stripped: output fulfil and still has normal char in text
-                out[-1] = COLORS['reset'] + wrap_char
-            elif count < width and pad:
```

Comment:
Thanks agin for your brilliant work, I only left some minor comments. Although new helper methods do not have unit test coverage, I believe it works perfectly because existing tests are untouched.

The `_markup_side_by_side()` method is horribly long, I wish I have time to simplify it..